Apps/Common/src/AspNetConfiguration/Modules/Observability.cs

```diff
@@ -270,29 +271,16 @@ private static LogEventLevel ExcludePaths(HttpContext ctx, Exception? ex, string
                 return LogEventLevel.Error;
             }
 
-            return Array.Exists(excludedPaths, path => IsWildcardMatch(ctx.Request.Path, path))
-                ? LogEventLevel.Verbose
-                : LogEventLevel.Information;
+            bool isWildcardMatch = Array.Exists(excludedPaths, path => IsWildcardMatch(ctx.Request.Path, path));
+            return isWildcardMatch ? LogEventLevel.Verbose : LogEventLevel.Information;
         }
 
         private static bool IsWildcardMatch(PathString requestPath, string path)
         {
-            if (!requestPath.HasValue)
-            {
-                return false;
-            }
-
-            if (path.EndsWith('*'))
-            {
-                return requestPath.Value!.StartsWith(path.Replace("*", string.Empty, StringComparison.InvariantCultureIgnoreCase), StringComparison.InvariantCultureIgnoreCase);
-            }
-
-            if (path.StartsWith('*'))
-            {
-                return requestPath.Value!.EndsWith(path.Replace("*", string.Empty, StringComparison.InvariantCultureIgnoreCase), StringComparison.InvariantCultureIgnoreCase);
-            }
-
-            return requestPath.Equals(path, StringComparison.InvariantCultureIgnoreCase);
+            return requestPath.HasValue &&
```

Comment:
Did not address all conditions.  Fixed issue in commit: 719f1bad539cdbcdd9253c9948391363fbf426b0